server/routes/debug.ts

```diff
@@ -76,35 +142,87 @@ export default function makeDebugRoutes(services: Services): Record<string, Requ
       if (toDateInput) {
         queryString.append('toDate', toDateInput)
       }
+      if (incidentType) {
+        queryString.append('incidentType', incidentType)
+      }
+      if (incidentStatuses) {
+        queryString.append('incidentStatuses', incidentStatuses)
+      }
+      const tableHeadUrlPrefix = `/incidents?${queryString}&`
+      if (sort) {
+        queryString.append('sort', sort)
+      }
+      if (order) {
+        queryString.append('order', order)
+      }
 
       const urlPrefix = `/incidents?${queryString}&`
-      const paginationParams = pagination(
-        pageNumber,
-        eventsResponse.totalPages,
-        urlPrefix,
-        'moj',
-        eventsResponse.totalElements,
-        eventsResponse.size,
-      )
-      const noFiltersSupplied = Boolean(!prisonId && !fromDate && !toDate)
 
-      const events = eventsResponse.content
-      const usernames = events.map(event => event.modifiedBy)
+      const noFiltersSupplied = Boolean(!prisonId && !fromDate && !toDate && !incidentType && !incidentStatuses)
+
+      const reports = reportsResponse.content
+      const usernames = reports.map(report => report.reportedBy)
       const usersLookup = await userService.getUsers(res.locals.systemToken, usernames)
       const prisonsLookup = await prisonApi.getPrisons()
+      const reportingOfficers = Object.values(usersLookup).map(user => ({
```

Comment:
i think most of the `Object.values(x)` can simply be `x` because they are arrays. you only need `Object.values` when you are iterating over an object.